ansible/devspace.yml

```diff
@@ -8,3 +8,15 @@
     - role: docker
     - role: versioncontrol-utils
     - role: devspace
+    - role: samba-client
+
+  tasks:
+
+    - name: Mount samba share
+      become: yes
+      mount:
+        fstype: cifs
+        name: /ome/data_repo
+        opts: "credentials=/etc/fstab.datarepo.creds,ro"
```

Comment:
It means your password can be in a file readable by root only, whilst `/etc/fstab` remains public.